tests/xapi-plugins/plugin_smartctl/test_smartctl.py

```diff
@@ -0,0 +1,19 @@
+# Requirements:
+# From --hosts parameter:
+# - host(A1): first XCP-ng host > 8.2.
```

Comment:
8.2.1 > 8.2, so this doesn't work. >= 8.3 would be more accurate.

Reply:
done